stock_reserve_sale/model/sale.py

```diff
@@ -108,23 +138,21 @@ def _get_line_rule(self):
                 ("route_id", "in", wh_route_ids),
             ]
 
-            rules = ProcurementRule.search(domain, order="route_sequence, sequence")
+            rules = StockRule.search(domain, order="route_sequence, sequence")
 
         if rules:
             return rules[0]
```

Comment:
You can use directly:
```suggestion
            return fields.first(rules)
```